### ISSUE TITLE
Pull on start with custom image

### ISSUE THREAD

**Issue by elmpp** [Shippable/support#2914]
I have tried my best to go through the console scripts section as well as scour the docs but can't seem to find a concrete answer to this.

I'm trying to get my custom image up and running but inevitably have some debugging to do. I'm amending my image, pushing to GCR but the changes do not seem to be propogated to subsequent shippable build runs. 

An example is here:
- https://app.shippable.com/runs/57c088f81fcce00e00958901/1/console
Basically, I've installled netstat and lsof to my image but they're not available (the GCR image has them).

Thanks.

**Comment by ryan-kimber** [comment]
In my experience, the clusters on GKE (or maybe it's the nodes) will cache a copy of the image based on the tag. If the tag doesn't change, then you'll still end up with the same image running.

I've run into this outside of Shippable too, just using the gcloud kubectl command line. 

The solution we're using is to create a unique tag when you push the image to your container registry. In our shippable.yml, that looks like this:

`build:
  ci:
   - pwd
  post_ci:
    - docker build -t nginx-for-gce:$BRANCH.$BUILD_NUMBER .
    - docker tag --force nginx-for-gce:$BRANCH.$BUILD_NUMBER us.gcr.io/formhero-io/nginx-for-gce:$BRANCH.$BUILD_NUMBER
    - docker push us.gcr.io/formhero-io/nginx-for-gce:$BRANCH.$BUILD_NUMBER

**Comment by elmpp** [comment]
I was actually referring to the custom image used for the build itself but will investigate GKE/GCR caching
